### DISCUSSION TITLE
Did I ruin my SB500??

### DISCUSSION THREAD

**Discussion by womeara6** [GerryDazoo/Slinger#669] [General]
Hi Gerry.  I had my SB500 running using slinger.  Unfortunately, I had to change my router and the router password.  I was unable to get my SB500 to bootup on the new network and I did a reset so I could change the ip address and network password.  When I rebooted I got the software update message and I let the software update take place.  the SB rebooted and took me through a setup series of pages where I was able to connect to the internet.  I verified that Slinger.server.exe  runs and after changing the config file to match the new ip address the SB connected to the server. The screen shows the server is waiting for the first stream.  The next set of pages on the Slingbox setup were designed to obtain the location of the provider and cable box.  This is where I am now stuck.  My cable company doesn't show up nor does it's set top box information.  At this point I can't go any further.  Did I ruin my SB500 with letting the software update?  Any thoughts on what to try next? Thanks for your help!!   Bill O.

**Comment by GerryDazoo** [comment]
That part that failed isn't used by Slinger it was only for sling to be able to give you a channel lineup. If you havn't gotten to the point of trying to setup the remote there's a good chance it's still intact. Just give the box a quick power cycle ( DO NOT do a factory reset) and you should be good to go.

**Comment by womeara6** [comment]
Gerry,  thanks for the quick reply.  On recycle, it tries the setup routine again and again. I can’t break out of it. The screen says “Slingbox Updated,”  Your Slingbox now has the latest software.  Using your Slingbox remote control, press the onscreen NEXT button to complete setup.

 

I can’t exit this setup routine and if I try to complete it, it connects to network ok and is assigned the static ip address that I gave it.  It requests Slingbox location by zip code and then takes me to Video Input where component is selected, “No Video Signal” is shown in the small video screen. The next screen says “Video Not Working” and suggests that I check connections. Screen allows you to “continue without video” but not recommended.   If I continue, I get “network error!” and it proceeds to TV Service Screen, with no provider given.  Next is Search for manufacturer of set top box and model. It offers a choose from list where again I get a “network error!”  The next screen “Device Manufacturer” offers a next option which does nothing.  

 

Slingbox_server.exe is running and it found the slingbox and logged in.  It’s last message is waiting for stream.

 

So the power up cycle doesn’t get me to the Slingbox to run normal.

 

I’m hoping you are right about the box being ok, but I can’t get it to see the connected TV.

 

Any help would be appreciated.

 

Thanks again,

Bill

 

 

Sent from my iPhone





On Dec 10, 2024, at 6:48 AM, GerryDazoo ***@***.***> wrote:

﻿

That part that failed isn't used by Slinger it was only for sling to be able to give you a channel lineup. If you havn't gotten to the point of trying to setup the remote there's a good chance it's still intact. Just give the box a quick power cycle ( DO NOT do a factory reset) and you should be good to go.

**Reply by GerryDazoo** [reply]
So, what happens when you try to start a stream? Make sure you have the proper video source set in the config.in file.
;500:        0=Composite       1=Component   2=HDMI

**Comment by womeara6** [comment]
Hi Gerry.  I’m sorry I was out this afternoon and couldn’t check on an answer to your question.  

 

Unfortunately, I am unable to stream anything. Further, I am to see the TV through the Slingbox with a straight hdmi connection from source thru Slingbox to TV.  This was not the case before.  I used to be able to see what was streaming on the TV with the signal source as the cable box.

 

My Slingbox 500 continues to boot into the setup routine and I can’t complete setup or break out of the routine.  While the start up for the Slingbox looks the same as before, after it starts it goes right to the screen saying the 

 

“Slingbox Updated,”  Your Slingbox now has the latest software.  Using your Slingbox remote control, press the onscreen NEXT button to complete setup.

 

I believe that the reset I did yesterday where I allowed the firmware to update has taken me out of the ability to even set up the video.  I’ve tried the config file for slingbox_server to use all three types of video sources with no success. The “picture in picture” on the video select screen says “no video.”  The only promising steps are that Slingbox_server does connect and the box is seen on the network with a static ip address.

 

By not being able to complete the setup of the box for the source it just Is not operating. Is there a way to rollback to prior firmware?  

 

I noticed that socelillyhammer ran into this same issue back with question SB500/350/M2/M1 - Reset #497 (SB500/350/M2/M1 - Reset · GerryDazoo/Slinger · Discussion #497 <https://github.com/GerryDazoo/Slinger/discussions/497> ) almost 2 years ago.  I didn’t see a resolution of the problem. 

 

I appreciate all your help.

 

Thanks,

Bill
